README.md

```diff
@@ -3,31 +3,88 @@
 ## Objectives
 
 1. Learn about requiring external code libraries, called gems, in your Ruby applications. 
-2. Learn how to use Bunlder and a Gemfile to manage gems and their dependencies in your applications. 
+2. Learn how to use Bundler and a Gemfile to manage gems and their dependencies in your applications. 
 
-## What is Bundler?
+
+
+### What are RubyGems?
+
+RubyGems is a package manager for the Ruby programming language that provides a standard format for distributing Ruby programs and libraries (in a self-contained format called a "gem"), a tool designed to easily manage the installation of gems, and a server for distributing them.
+
+#### How to find a gem?
+
+One option is always to google. Lets assume we are looking for a gem to send emails. We could google `ruby gem to send emails`. One of the first result is a link to `https://www.ruby-toolbox.com/categories/e_mail`.
+
+Another option is to go to `https://rubygems.org/` and enter `mail` in the search bar.
+
+#### How to choose the right gem?
+
+Lets assume we googled and now we are on the `ruby-toolbox.com` website, which shows a list of all the gems that help with sending mails. Which one is the best? How to choose from those many options?
+
+At the top of the page you will the the most popular gem listed - `ActionMailer`, `Mail` and `Pony `are the top 3.
+
+![](http://readme-pics.s3.amazonaws.com/the-ruby-toolbox-most-popular.png)
+
+Which one should I use for my application? 
+
+First of all you, if you have a rails application you should use gems build for rails, like the `actionmailer` gem. How do we know it is a rails application? If you look at the Github link, you'll notice this gems lives under `rails/rails` repository.
+
+![](http://readme-pics.s3.amazonaws.com/actionmailer.png)
+
+![](http://readme-pics.s3.amazonaws.com/mail.png)
+
+We don't have a rails application, so we need to decide between the `mail` and `pony` gem. The easiest way is to this by inspecting the total fork count, or by going to the Github repository. Some questions that will help with making your decision: 
+
+* How many people forked the repo? 
+* How many are contributed to the repo?
+* How many open issues does the repo have? 
+* Does the repo have a test suite?
+
+#### How to install a gem?
+
+We are going to use the `mail` gem. One way of installing it would be to run `gem install mail` in your terminal.
+
+Assuming you have a application the other way would be to add `gem 'mail', '~> 2.6', '>= 2.6.3'` (which is the current version of the gem) to your `GEMFILE`. This makes sure who ever uses your application would also use the same version of the gem.
```

Comment:
Before giving the GEMFILE example. Show how to include gem with just the standard `require`